v1.0/Obsolete/Part 2: File Format Policies.md

```diff
@@ -40,7 +40,7 @@ The following table describes the requirements for data quality when preserving
 | --- | --- | --- |
 | ODS_1 | Password protection | The file **MUST NOT** have read or write password protection. |
 | ODS_2 | Standard compliance | The file **MUST** comply with the standard “OASIS Open Document Format for Office Applications (OpenDocument) v1.3”. |
-| ODS_3 | Extension | The file **MUST** have extension “.fods”, “.ods” or “ots”. |
+| ODS_3 | Extension | The file **MUST** have extension “.fods”, “.ods” or “.ots”. |
```

Comment:
More a musing than a direct review comment. Does this mean that we won't support templates at all? Might be a missed opportunity, given that there are no differences in many ways. Put another way, validating a template might be useful functionality. It would still be possible to disallow them to abide by organisational policy.